modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm

```diff
@@ -113,14 +114,12 @@
 
 /obj/structure/closet/secure_closet/security_medic/PopulateContents()
 	..()
-	new /obj/item/clothing/suit/toggle/labcoat/security_medic(src)
-	new /obj/item/clothing/suit/hazardvest/security_medic(src)
+	new /obj/item/clothing/suit/toggle/labcoat/security_medic/blue(src)
+	new /obj/item/clothing/suit/hazardvest/security_medic/blue(src)
 	new /obj/item/clothing/head/helmet/sec/peacekeeper/security_medic(src)
-	new /obj/item/clothing/under/rank/security/peacekeeper/tactical(src)
 	new /obj/item/radio/headset/headset_medsec(src)
 	new /obj/item/storage/medkit/emergency(src)
 	new /obj/item/clothing/suit/straight_jacket(src)
 	new /obj/item/storage/belt/medical(src)
 	new /obj/item/storage/belt/security/medic/full(src)
 	new /obj/item/clothing/under/rank/medical/doctor/red(src)
```

Comment:
```suggestion
	new /obj/item/clothing/under/rank/medical/doctor/red(src)
	new /obj/item/clothing/under/rank/security/peacekeeper/security_medic/old
```
It can stay while security medic has a full overhaul